### DISCUSSION TITLE
Unable to pull private keys

### DISCUSSION THREAD

**Discussion by alaskanrabb** [bubuntux/nordlynx#74] [Q&A]
I am attempting to pull my private key and I get this.... anyone else still able to pull?

Unable to access interface: Protocol not supported

Here is your private Key:

**Comment by aurelized** [comment]
i am facing the same issue. Did you manage to fix it?

**Comment by alaskanrabb** [comment]
I did, I setup an omv 6 server that had dependencies that bubuntux’s older
nord container needed to connect and pull the key.  I tried to shoehorn in
the dependencies on my older setup but I was migrating to a new server and
it worked fine.

**Reply by aurelized** [reply]
thank you for your prompt reply. Let's hope I find another more simple way then!

**Comment by aurelized** [comment]
in the end I did more or less the same thing as you: i installed ubuntu on a VM, and followed this 

https://gist.github.com/bluewalk/7b3db071c488c82c604baf76a42eaad3

and this

https://gist.github.com/bluewalk/7b3db071c488c82c604baf76a42eaad3
